docs/en/pid-eaa-issuance.rst

```diff
@@ -921,7 +998,47 @@ If the Credential Request is invalid, the PID/(Q)EAA Provider MUST return an err
 .. literalinclude:: ../../examples/credential-error.json
   :language: JSON  
 
+Notification endpoint
+---------------------
+
+The Notification Endpoint is used by the Wallet to notify the PID/(Q)EAA Provider of certain events for issued Credentials, such as if the Credential was successfully stored in the Wallet Instance or in case of unsuccessful Credential issuance caused by a user action.
```

Comment:
```suggestion
The Notification Endpoint is used by the Wallet to notify the PID/(Q)EAA Provider of certain events for issued Credentials, such as if the Credential was successfully stored in the Wallet Instance or in case of unsuccessful Credential issuance caused by a User action.
```